### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by Thisismydigitalself** [eyal0/OctoPrint-PrintTimeGenius#149]
Hi Eyal, 

Any chance for your genius plugin (Thank u!) to command octoprint to open this pull down window once PTG is done doing its thingy?.. 

![ice_screenshot_20190824-105058](https://user-images.githubusercontent.com/30949968/63635132-9ed1ee00-c667-11e9-97ae-908968a4cfaa.png)

**Comment by eyal0** [comment]
I don't think that this is something that could or should be part of PTG.  I think that this should be a whole different plugin, if that's something that is needed.

Why do you need it?

**Comment by Thisismydigitalself** [comment]
Hi,
When your Plugin finishes its calculation and yellow star stops blinking the pull down menu stays closed and you don't see the results. you need to open the window to see those. the upper octoprint information above is updated only after starting the print job. i don't see a reason why your plug should not present the results when they are ready.

This is when calculation is done:
![ice_screenshot_20190904-161655](https://user-images.githubusercontent.com/30949968/64258608-37058800-cf30-11e9-8f20-8ab8dfcc2d71.png)

This is when i manually click on this arrow down/open file tab and see the results:
![ice_screenshot_20190904-161709](https://user-images.githubusercontent.com/30949968/64258619-3c62d280-cf30-11e9-9afc-77a136ceaed8.png)
